### PR TITLE
Adding Contact Routing the way Sensu Enterprise does it

### PR DESCRIPTION
**Pull request by stevenviola**
## Pull Request Checklist

**Is this in reference to an existing issue?** [Yes](https://github.com/sensu-plugins/sensu-plugins-mailer/issues/23)

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass 

#### New Plugins

- [x] Tests

- [x] Add the plugin to the README

- [x] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose
Add ability to configure particular checks or clients to e-mail different contacts, similar to the existing [Sensu Enterprise contact routing](https://sensuapp.org/docs/0.26/enterprise/contact-routing.html) usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,40 @@ client config called "template", the mailer handler config, default.
...
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      }
+    }
+  }
+} 
+```
+
+Then, in a check defination, you can specify a contact or an array of contacts which should be notified by e-mail:
```
Comment (EvaSDK) [inline README.md]: defination → definition I guess

Comment (stevenviola) [inline README.md]: I've gone ahead and fixed this in bde964a

**Comment by majormoses** [comment]
I have never used enterprise version, @cwjohnston @eheydrick could you review this?

**Comment by eheydrick** [comment]
Looks good to me based on eyeballing the code.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-mailer/versions/1.2.0

**Pull request merged**